src/constants.ts

```diff
@@ -2,8 +2,15 @@ import { ethers } from 'ethers'
 
 export const CHI = ethers.utils.formatBytes32String('CHI').slice(0, 14)
 export const RATE = ethers.utils.formatBytes32String('RATE').slice(0, 14)
+
 export const DAI = ethers.utils.formatBytes32String('01').slice(0, 14)
 export const USDC = ethers.utils.formatBytes32String('02').slice(0, 14)
 export const ETH = ethers.utils.formatBytes32String('00').slice(0, 14)
 export const STETH = ethers.utils.formatBytes32String('05').slice(0, 14)
 export const WSTETH = ethers.utils.formatBytes32String('06').slice(0, 14)
+
+//yearn vaults 1xx
+export const YVETH = ethers.utils.formatBytes32String('100').slice(0, 14)
+export const YVDAI = ethers.utils.formatBytes32String('101').slice(0, 14)
```

Comment:
second question, just confirming, would you like me to change these yearn vault constants to the next consecutive ordinal numbers as you suggested in a recent code review comment?

If I did that, I would not include yvSTETH or yvETH in this pr.  and make yvDai and yvUsdc be "08" and "09" after ens

I would add yvUSDC and yvDai as "09" and "10" respectively

Reply:
Yes please, let's stick to consecutive ordinals.